src/game/client/neo/ui/neo_hud_message.cpp

```diff
@@ -178,3 +192,64 @@ void CNEOHud_Message::HideMessage()
     m_szMessage[0] = L'\0';
     m_bShouldDraw = false;
 }
+
+std::wstring CNEOHud_Message::ProcessKeyBinds(const wchar_t* rawmessage) const
```

Comment:
We should ideally be able to handle the standard `"%%"` percent sign escape case, so that we have support for inputs like:
```
"Tutorial_Bind_Use"			"Drink your 100%% orange juice with %+use%"
```
so something like
```diff
--- a/src/game/client/neo/ui/neo_hud_message.cpp
+++ b/src/game/client/neo/ui/neo_hud_message.cpp
@@ -217,6 +217,13 @@ std::wstring CNEOHud_Message::ProcessKeyBinds(const wchar_t* rawmessage) const
             break;
         }

+        if (start + 1 == end)
+        {
+            outputMessage.append(L"%");
+            pos = end + 1;
+            continue;
+        }
+
         std::wstring unicodeBinding = inputMessage.substr(start + 1, end - start - 1); // Extract the command

         char binding[64];
```